webbpsf/trending.py

```diff
@@ -1038,24 +1471,31 @@ def monthly_trending_plot(year, month, verbose=True, instrument='NIRCam', filter
         Table of available OPDs, Default None: as returned by retrieve_mast_opd_table()
     """
 
-    
     def vprint(*text):
-        if verbose: print(*text)
+        if verbose:
+            print(*text)
 
     start_date, end_date = get_month_start_end(year, month)
-            
+
     # Look up wavefront sensing and mirror move corrections for that month
     if pid:
         pid_dates = get_dates_for_pid(pid)
         if pid_dates is not None:
             pid_dates = pid_dates[(pid_dates >= start_date) & (pid_dates <= end_date)]
         else:
             pid = None
-        
-    if opdtable is None:
-        opdtable = get_opdtable_for_month(year, month)
-    else:
-        opdtable = check_colnames(opdtable)
+
```

Comment:
logic change here to account for actual filtering if we pass our own opdtable, and also to handle errors